lib/wallet/wallet.js

```diff
@@ -1559,6 +1559,87 @@ class Wallet extends EventEmitter {
     }
   }
 
+  /**
+   * Make a batch open MTX.
+   * @param {Array} names
+   * @param {Number|String} acct
+   * @param {Boolean} force
+   * @returns {MTX}
+   */
+
+  async makeBatchOpen(names, force, acct) {
+    // TODO roza - verify this method is supported
+    assert(Array.isArray(names));
+    assert(typeof force === 'boolean');
+    assert(acct >>> 0 === acct || typeof acct === 'string');
+
+    const errors = [];
+    const mtx = new MTX();
+
+    for (const name of names) {
+      if (!rules.verifyName(name)) {
+        errors.push({name: name, error: 'Invalid name.'});
```

Comment:
The key `error` makes this sound like a typed object, could you call it `message` or `errorMessage`?